admin/horace_mex_mpi.m

```diff
@@ -5,7 +5,7 @@
 % Manually modify this script to specify the mpi libraries location in your
 % system.
 %
-use_her_mpich = false;
+use_her_mpich = true;
```

Comment:
Why are we changing defaults here?

Reply:
because I was trying to build `cpp_communicator` using the library, we provide with `Herbert` and it worked. Probably it is a bit easier to build `cpp_communicator`  with the libraties we provide with Herbert, than install external MPI on a system and then build `cpp_communicator` (`use_her_mpic=false` -- look for MPI installed on a system)

There are actually little difference between these two options as compiling `cpp_communicator` is a simpler task than making it to work, which I will try to explain in a fresh ticket.

Reply:
As from Slack earlier: two things. One, if it needs that much explanation, then it should be documented in the code. Please add a substantial comment saying what is going on there. Two, I don't understand your explanation. It sounds more as if there was a temporary glitch in the build process rather than something fundamental that requires this change. Please can you write a paragraph for comments and run them by us. If need be we can use the sprint review slot tomorrow to discuss it.

Reply:
There are generally not advisable to describe in words the thing the code is doing. You should comment on meanings and describe intentions. This requests violates this principle, but it is not difficult to add what the code is doing adding couple of sentences.